manager/controllers/app/plotter_generator.go

```diff
@@ -59,7 +59,7 @@ func (p *PlotterGenerator) GetCopyDestination(item DataInfo, destinationInterfac
 
 	// S3 endpoint should not include the url scheme only the host name
 	// thus ignoring it if such exists.
-	url, err := url.Parse(bucket.Endpoint)
+	url, err := url.Parse(bucket.Endpoint[0])
```

Comment:
bucket should include a single endpoint and a single region.

Reply:
Changed